src/ui/ToggleEditor.tsx

```diff
@@ -503,9 +514,7 @@ class ToggleEditor extends Component<ToggleEditorProps, ToggleEditorState> {
         languageId={this.props.language.id}
         options={{...defaultOptions, ...this.props.options}}
         showDialog={this.showDialog}
-        closeDialog={this.closeDialog}
         toolbarRef={this.toolbarRef}
-        debugHistory={this.props.debuggingLog?.history}
      />
```

Comment:
I've removed `closeDialog` and `debugHistory` since they didn't appear to be used anywhere.

Reply:
Wait, are you sure `closeDialog` isn't used? Aren't we passing this function from the ToggleEditor into the Dialog component?

Reply:
`this.closeDialog` is passed into the `Dialog` component in the `ToggledEditor.render` function. This is the `ToggleEditor.renderBlocks` function. The block editor doesn't render the dialog, so it doesn't have a closeDialog prop.